tests/Rector/Class_/YieldDataProviderRector/Fixture/should_not_remove_comments.php.inc

```diff
@@ -38,17 +40,18 @@ use PHPUnit\Framework\TestCase;
 final class ShouldNotRemoveComments extends TestCase
 {
     /**
-     * @dataProvider provideData()
+     * some comment
+     *
+     * @dataProvider provideDataSomething()
      */
-    /**
-     * @dataProvider provideData()
-     */
-    public function test(string $filePath): void
+    public function testSomething(string $filePath): void
     {
     }
 
-    public function provideData(): \Iterator
+    public function provideDataSomething(): \Iterator
     {
+        // Let's pretend that this comment is extremely important and meaningful.
+        // It should not be removed by Rector.
```

Comment:
@TomasVotruba this need to be included in "after" part to demonstrate the fixture goal